src/components/calcite-date-month-header/calcite-date-month-header.tsx

```diff
@@ -90,14 +90,14 @@ export class CalciteDateMonthHeader {
       <Host dir={dir}>
         <div aria-hidden="true" class="header">
           <a
-            aria-disabled={nextMonthDate.getMonth() === activeMonth}
+            aria-disabled={prevMonthDate.getMonth() === activeMonth}
             aria-label={this.intlNextMonth}
             class="chevron"
             href="#"
             onClick={(e) => this.handleArrowClick(e, prevMonthDate)}
             onKeyDown={(e) => this.handleKeyDown(e, prevMonthDate)}
             role="button"
-            tabindex="0"
+            tabindex={prevMonthDate.getMonth() === activeMonth ? -1 : 0}
           >
             <calcite-icon dir={dir} icon="chevron-left" mirrored scale={iconScale} />
```

Comment:
As @paulcpederson mentioned - this just recently changed to `flip-rtl` in this PR that was merged: https://github.com/Esri/calcite-components/pull/1132